src/Pos/Launcher/Runner.hs

```diff
@@ -422,7 +422,9 @@ bracketDHTInstance BaseParams {..} action = bracket acquire release action
     instConfig =
         KademliaDHTInstanceConfig
         { kdcKey = bpDHTKey
-        , kdcHost = maybe "0.0.0.0" fst bpNetworkAddress
+        --, kdcHost = maybe "0.0.0.0" fst bpNetworkAddress
+        , kdcHost = "0.0.0.0" -- Amazon servers have different ifconfig (local) ip and public ip
+                              -- so we can't bind on public directly.
```

Comment:
The right solution here is to have a separate Kademlia configuration. I have this in my branch work to abstract the discovery mechanism.